[RUMS-4743] Expo: added EAS updates documentation (#31482)
DataDog/documentation: content/en/real_user_monitoring/error_tracking/mobile/expo.md

module.exports = config;
```

> ### Uploading sourcemaps for EAS updates

Reviewer: ```suggestion
### Upload sourcemaps for EAS updates
```

When creating an [EAS update](https://docs.expo.dev/eas-update/introduction/), you’ll also need to upload the corresponding sourcemaps to Datadog. This ensures accurate error tracking and symbolication.

#### 1. Create the EAS update

Run the update command as usual:

```
eas update --channel [channel-name] --message "[message]"
```

> This will generate a `dist` folder at the root of your project.

Reviewer: ```suggestion
This generates a `dist` folder at the root of your project.
```

> ---
> 

Reviewer: ```suggestion
```

#### 2. Locate the sourcemaps

> Inside the `dist` folder, you’ll find the generated bundles and sourcemaps:

Reviewer: ```suggestion
Inside the `dist` folder, find the generated bundles and sourcemaps:
```

* Example paths:

  * `./dist/_expo/static/js/ios`
  * `./dist/_expo/static/js/android`

* File pairs you may see:

  * Hermes: `.hbc` (bundle) and `.hbc.map` (sourcemap)
  * JSC: `.js` or `.jsbundle` (bundle) and `*.map` (sourcemap)

> ---
> 

Reviewer: ```suggestion
```
